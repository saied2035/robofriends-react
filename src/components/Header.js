import {memo} from 'react';

const Header = () => { 
	return <h1 className="f2 ma2">RoboFriends</h1>
  }	



export default memo(Header);