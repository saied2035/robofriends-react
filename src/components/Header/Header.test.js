import {shallow} from 'enzyme'
import Header from './Header'

it("render Header",() => {
        expect(shallow(<Header />)).toMatchSnapshot()
})