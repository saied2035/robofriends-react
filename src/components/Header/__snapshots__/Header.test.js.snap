// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Header 1`] = `
<h1
  className="f2 ma2"
>
  RoboFriends
</h1>
`;
